nifi-marklogic-services/src/main/java/org/apache/nifi/marklogic/controller/DefaultMarkLogicDatabaseClientService.java

```diff
@@ -199,14 +200,12 @@ protected DatabaseClientConfig buildDatabaseClientConfig(ConfigurationContext co
 
         final SSLContextService sslService = context.getProperty(SSL_CONTEXT_SERVICE).asControllerService(SSLContextService.class);
         if(sslService != null) {
-            final SSLContextService.ClientAuth clientAuth = determineClientAuth(context);
+            final ClientAuth clientAuth = determineClientAuth(context);
+            getLogger().info("Configuring SSL connection; client authentication: " + clientAuth);
```

Comment:
I don't know enough about the ClientAuth class - are we confident this won't print any sensitive data?

Reply:
It's an enum class with properties of "type" and "description"; the "toString" is actually pretty helpful as it includes both. Doesn't include any sensitive data, as it's just a type of authentication, not the authentication itself.